docs/sdk/quickstart-by-language/quickstart-nodejs.mdx

```diff
@@ -0,0 +1,156 @@
+---
+sidebar_position: 1
+title: Permit NodeJS SDK
+---
+
+Working with the Permit.io Node.js SDK enables developers to seamlessly integrate robust authorization management into their
+applications. This SDK simplifies the implementation of fine-grained access controls, whether you're managing user roles with
+RBAC, policies with ABAC, or relationships with ReBAC.
+
+:::note Prerequisites:
+This tutorial is for users who have already set up at least one policy in Permit.io. If you haven't done so yet,
+please refer to [this guide](/quickstart) first.
+:::
+
+## Adding the Permit.io Node SDK to your project
+
+### 1. Install the Permit.io SDK
+
+To start, you need to install the Permit.io SDK in your project. Open your terminal and run the following command:
+
+```sh
+npm install permitio
+```
+
+This will add the Permit.io SDK to your project, allowing you to integrate authorization management seamlessly.
+
+### 2. Import the SDK into your code
+
+After installing the SDK, you need to import it into your application.
+
+### 3. Create a new Instance of the SDK
+
+Next, create a new instance of the Permit SDK. You'll need a secret API key, which you can obtain by following the instructions
+provided in the setup section of the Permit.io documentation.
+
+```javascript
+const permit = new Permit({
+  token: "[YOUR_API_KEY]",
+  pdp: "http://localhost:7766",
+});
+```
+
+Replace "YOUR_API_KEY" with the actual API key you obtained. Find instructions how to fetch your API key here.
+
+## The fundamental functionality - Checking for permissions
+
+`permit.check()` is the fundamental SDK function that controls the enforcement in your application level code. It acts as a guard of who
+can do what, based on the policy that you defined.
+
+Each user, once his identity is verified by the authentication provider, gets issued a session. During this session, his unique id
+obtained from the JWT (JSON Web Token), which is synced to Permit, maps to a specific role within the Policy.
+
+Each role has a set of rules that were pre-defined, which a policy engine can understand, evaluate, and therefore return if the user
+is permitted or not, based on the user id, action and resource passed into the SDK function.
+
+You can run a permission check with permit.check(), passing in 3 arguments:
+
+- `user`: a unique string id that identifies the user doing the action (and the currently logged in user).
+- `action`: the action performed.
+- `resource`: the resource (object) the action is performed on.
+
+In the following example we are checking that a user with the unique id `john@smith.com` can `read` a `document`.
+
+```javascript
+const permitted = await permit.check("john@permit.io", "read", "document");
+
+if (permitted) {
+  console.log("John is PERMITTED to read a document");
+} else {
+  console.log("John is NOT PERMITTED to read a document");
+}
+```
+
+## Other basic functionality with the SDK
```

Comment:
As this page shouldn't be a remix of the **Permit Product Quickstart** tutorial, I think this `fundamental` and `basic` should be "flatly" categorized into the various main functions of the **Permit client**:
- Check (enforcement)
- API (configuration)
Beside of that, we should also dedicate a section for unique SDK functions, such as `elements` that we have in the JS SDK.